### ISSUE TITLE
"Quickinstall" Collab is complaining of a warning when installing deepchecks

### ISSUE THREAD

**Issue by AlmogBaku** [deepchecks/deepchecks#418]
```!pip install deepchecks```

```
...
      Successfully uninstalled plotly-4.4.1
  Attempting uninstall: matplotlib
    Found existing installation: matplotlib 3.2.2
    Uninstalling matplotlib-3.2.2:
      Successfully uninstalled matplotlib-3.2.2
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
albumentations 0.1.12 requires imgaug<0.2.7,>=0.2.5, but you have imgaug 0.2.9 which is incompatible.
Successfully installed category-encoders-2.3.0 deepchecks-0.1.1 fonttools-4.28.5 matplotlib-3.5.1 plotly-5.5.0 scipy-1.7.3 tenacity-8.0.1
WARNING: The following packages were previously imported in this runtime:
  [matplotlib,mpl_toolkits]
You must restart the runtime in order to use newly installed versions.
```

**Comment by ItayGabbay** [comment]
@AlmogBaku Thanks for the issue!

This specifically happens on the Google Colab GPU kernel. 
After investigation, I'm closing this issue since it's related to Google Colab env issues, and it can be simply fixed by restarting the kernel (Which is common after a package installation)
